localstripe/resources.py

```diff
@@ -244,6 +246,76 @@ def do_expand(path, obj):
         return obj
 
 
+class Balance(object):
+    object = 'balance'
+
+    def __init__(self, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        currencies = ['eur']
+
+        self.object = self.object
+        self.livemode = False
+        self.type = 'balance'
+        self.connect_reserved = []
+        self.available = [
+            {
+                'amount': 2000,
+                'currency': currency,
+                'source_types': {
+                    'card': 2000
+                }
+            }
+            for currency in currencies
```

Comment:
If there is only one currency maybe this part can be simplified?
A PR is perfect not when there is nothing to add, but when there is nothing to remove :P